float32[3] attitude_euler_angle	    # Attitude/direction of the mount as euler angles in rad
